### PR TITLE
Only use yarn, fix comments

### LINKED ISSUE #1925: Fix send_nft() doc comments
Currently it's talking about native tokens

### LINKED ISSUE #1949: Use only yarn in bindings
Remove npm support

### PR DESCRIPTION
**Pull request by Thoralf-M**
# Description of change

Only use yarn so we don't have multiple lock files. Changed the examples installation to use `yarn link`, because otherwise it always failed with 
```
error: current package believes it's in a workspace when it's not:
current:   /iota-sdk/bindings/nodejs/examples/node_modules/@iota/sdk/Cargo.toml
workspace: /iota-sdk/Cargo.toml
```
Also updated wrong comments.

## Links to any relevant issues

Closes #1949 
Closes #1925 

## Type of change

- Enhancement (a non-breaking change which adds functionality)
- Documentation Fix

## How the change has been tested

Running installation commands and an example afterwards

### CODE REVIEW

**Review by thibault-martinez — COMMENTED** [review]
Can you check the READMEs pls? I think they may reference npm

File: bindings/wasm/README.md [hunk 1]
```diff
@@ -31,13 +31,7 @@ environments.**
...
-To install the library from your package manager of choice, you only need to run the following:
-
-#### npm
-
-```bash
-npm i @iota/sdk-wasm
-```
+To install the library with yarn, you only need to run the following:
 
 #### yarn
```
Comment (thibault-martinez) [inline bindings/wasm/README.md]: could remove this

File: bindings/nodejs/README.md [hunk 2]
```diff
@@ -44,13 +43,7 @@ an environment variable `RUSTFLAGS` to `-C target-feature=+crt-static`.
...
-To install the library from your package manager of choice, you only need to run the following:
-
-#### npm
-
-```sh
-npm i @iota/sdk
-```
+To install the library with yarn, you only need to run the following:
 
 #### Yarn:
```
Comment (thibault-martinez) [inline bindings/nodejs/README.md]: same

**Review by thibault-martinez — DISMISSED** [review]

File: sdk/src/wallet/account/operations/transaction/high_level/minting/create_native_token.rs [hunk 3]
```diff
@@ -91,8 +91,8 @@ where
 {
     /// Creates a new foundry output with minted native tokens.
     ///
-    /// Calls [Account::send_outputs()](crate::wallet::Account::send_outputs) internally, the options may define the
-    /// remainder value strategy or custom inputs. Note that addresses need to be bech32-encoded.
+    /// Calls [Account::prepare_transaction()](crate::wallet::Account::prepare_transaction) internally, the options may
+    /// define the remainder value strategy or custom inputs. Note that addresses need to be bech32-encoded.
```
Comment (Alex6323) [inline sdk/src/wallet/account/operations/transaction/high_level/minting/create_native_token.rs]: Cannot select the lines below, but the params aren't correct either. Also ... could you remove the trailing comma in the arg list in L104? xD

Comment (Thoralf-M) [reply sdk/src/wallet/account/operations/transaction/high_level/minting/create_native_token.rs]: What's wrong with the params?

File: bindings/nodejs/README.md [hunk 4]
```diff
@@ -8,7 +8,6 @@
     - [Windows](#windows)
   - [Getting Started](#getting-started)
     - [Installation Using a Package Manager](#installation-using-a-package-manager)
-      - [npm](#npm)
       - [Yarn](#yarn)
```
Comment (thibault-martinez) [inline bindings/nodejs/README.md]: Need to remove that now :trollface:

File: bindings/nodejs/README.md [hunk 5]
```diff
@@ -159,21 +150,21 @@ You can find the API reference for the Node.js bindings in the
...
 
-### `npm run test` or `yarn test`
+### `yarn test`
 
 Runs the unit tests by calling `cargo test`. You can learn more
 about [adding tests to your Rust code](https://doc.rust-lang.org/book/ch11-01-writing-tests.html) from
 the [Rust book](https://doc.rust-lang.org/book/).
 
-### `npm run create-api-docs` or `yarn create-api-docs`
+### `yarn create-api-docs`
```
Comment (Alex6323) [inline bindings/nodejs/README.md:166]: putting this comment here (also nitpick^1000)
L191 -> ... which informs the `yarn` command.

File: sdk/src/wallet/account/operations/transaction/high_level/minting/create_native_token.rs (hunk 3 above)
Comment (Alex6323) [reply sdk/src/wallet/account/operations/transaction/high_level/minting/create_native_token.rs]: nothing ... forgot we're on develop, sorry

File: bindings/nodejs/README.md (hunk 5 above)
Comment (Thoralf-M) [reply bindings/nodejs/README.md:166]: This section is so useless, but updated

File: bindings/nodejs/README.md (hunk 4 above)
Comment (Thoralf-M) [reply bindings/nodejs/README.md]: Done

**Review by thibault-martinez — APPROVED** [review]

**Review by Alex6323 — APPROVED** [review]

**Pull request merged**